src/SSL/Entities/ValidationResult.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace UKFast\SDK\SSL\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property bool $result
+ * @property array $altHosts
+ * @property int $expiryTimestamp
```

Comment:
It would be nice to make these consistent with the rest of the SDK, maybe `domains` & `expiresAt`?